### PR TITLE
Displaying error message on course-edit page in case of error.

### PR DESCRIPTION
**Pull request by awais786**
https://openedx.atlassian.net/browse/ECOM-7666

screen shot 2017-04-11 at 2 47 19 pm

### CODE REVIEW

File: course_discovery/apps/publisher/views.py [hunk 1]
```diff
@@ -398,6 +398,10 @@ def form_valid(self, form):
         messages.success(self.request, _('Course  updated successfully.'))
         return HttpResponseRedirect(self.get_success_url())
 
+    def form_invalid(self, form):
+        messages.error(self.request, _('Please fill all required fields.'))
```
Comment (waheedahmed) [inline course_discovery/apps/publisher/views.py]: Translations does not have this message, I think you need to re-generate.

Comment (waheedahmed) [reply course_discovery/apps/publisher/views.py]: Also quality will fail, you need to disable pylint no-member for this `# pylint: disable=no-member`

**Review by waheedahmed — APPROVED** [review]
Couple of comments, 👍 once addressed.

**Comment by srpearce** [comment]
@awais786 Please change the error message to the following:

Make sure to provide all required information.

**Pull request merged**